### ISSUE TITLE
Force National Language is lost after screen orientation

### ISSUE THREAD

**Issue by GeoHigh125** [geopaparazzi/geopaparazzi#543]
After my device has switched between the portrait and landscape screen modes , I lose always the Force National Language setting for the Dutch (NL) language and are back in the native English setting.
I'm using Redmi Note 4X with Android vesion 6.0

**Comment by moovida** [comment]
That is really odd. Does this happen from any view?

**Comment by GeoHigh125** [comment]
Yes, but it happens only on my phone. This behavior is not happening on my tablet. May be it something initiating by the phone.

**Comment by moovida** [comment]
I am not able to reproduce this. Can someone confirm this happens?
